services/settings-service/src/main/java/com/amazon/aws/partners/saasfactory/saasboost/SettingsServiceDAL.java

```diff
@@ -164,6 +172,20 @@ public List<Map<String, Object>> rdsOptions() {
         return orderableOptionsByRegion;
     }
 
+    public List<CertificateSummary> acmCertificateOptions() {
```

Comment:
I'm assuming by leaving out `maxItems` on the request it will default to 10,000? Probably don't need to implement a while or do/while loop here with `nextToken` but I will point it out for completeness.

Reply:
You're right, correctness requires we loop on `nextToken`. Good catch!